.github/workflows/ci.yml

```diff
@@ -44,3 +44,22 @@ jobs:
       - name: Circular dependency check
         continue-on-error: true
         run: pnpm run madge
+
+      - name: Install /test dependencies
+        run: pnpm install --ignore-workspace
+        working-directory: ./test
+
+      - name: Install Playwright Browsers
+        run: pnpm exec playwright install --with-deps
```

Comment:
Something to be aware about, if we end up using snapshots to test diffs, running playwright in the github runner vs. locally will actually cause snapshots to be different. The reason being is that this Github runner is using `linux` and local machine will be using `macos` unless you configure the Github running to use the `macos` runner. The snapshot diffs will be different in that case, so maybe we can write tests to avoid snapshotting.

Reply:
Hmm. Interesting.

What do you mean by snapshotting? I usually think of snapshot testing as diffing some sort of media( json, image, etc). 

I am thinking of just diffing attributes in the DOM, like CSS outputs. I think this should be consistent between os.

Reply:
https://playwright.dev/docs/test-snapshots

It's basically visual regression testing, which again, I don't think we'll use here.

Reply:
Why would the snapshots be different in different operating systems? I would expect it to be possible to have consistent snapshots regardless of the OS.

Reply:
Snapshot files have the os as part of their name that they compare against:
![Screenshot 2024-01-26 at 14 35 06](https://github.com/makeswift/makeswift/assets/10539418/610f00eb-09eb-42cd-b84a-d9448de413de)

You'll have to spin up a docker container to update your snapshots.

Reply:
This is good to know in case we go down that route.